staging/src/k8s.io/cli-runtime/pkg/genericclioptions/record_flags.go

```diff
@@ -93,6 +93,7 @@ func (f *RecordFlags) AddFlags(cmd *cobra.Command) {
 
 	if f.Record != nil {
 		cmd.Flags().BoolVar(f.Record, "record", *f.Record, "Record current kubectl command in the resource annotation. If set to false, do not record the command. If set to true, record the command. If not set, default to updating the existing annotation value only if one already exists.")
+		cmd.Flags().MarkDeprecated("record", "--record will be removed in the future")
```

Comment:
```suggestion
		cmd.Flags().MarkDeprecated("record", "--record will be removed in a future Kubernetes release")
```
?

Reply:
That's consistent with all the rest of deprecation messages so I left is as is.